app/controllers/roles_controller.rb

```diff
@@ -225,4 +230,38 @@ def count_only?
   def count_payload(dataset)
     { count: dataset.count }
   end
+
+  def audit_memberships_success(filter)
+    additional_params = %i[account count search kind filter]
+    options = params.permit(*additional_params)
+                    .slice(*additional_params).to_h.symbolize_keys
```

Comment:
We should also strip the extra `slice` here:

```suggestion
                    .slice(*additional_params).to_h.symbolize_keys
```
```suggestion
                    .to_h.symbolize_keys
```